### PR TITLE
Use gm instead of phantom to create pngs

### PR DESCRIPTION
**Pull request by FezVrasta**
This should speed up a lot the conversion from svg to png.

Imagemagick is required on the host machine, I'm not sure if the docker image you use has it?

ping @espadrine

### CODE REVIEW

File: svg-to-img.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
+var Rsvg = require('librsvg-prebuilt').Rsvg;
```
Comment (espadrine) [inline svg-to-img.js]: Note that I haven't tested librsvg; although it seems to rely on cairo, which we relied on in the past for font size computation, and that dependency was a pain point to contributions and sysadmin.

**Comment by FezVrasta** [comment]
Thanks for the feedback, I'm going to fix the transparency problem ASAP

**Comment by FezVrasta** [comment]
Transparency problem has been resolved.

**Comment by FezVrasta** [comment]
So, I tried the code using Docker, and the output font is the expected one:

image

So, looks like a problem with macOS and my setup. I'd say it's good to go then!

**Comment by espadrine** [comment]
I do get font issues on my Linux machine as well, which doesn't bode well. However, like I mentioned, PNG images are never ideal anyway, as more and more screens double their pixel count. I'll put this live; if the result is unreadable, I will revert this and we'll try to go the inkscape route. I'm fairly optimistic about this though!

There is one modification that I will make: `.density(90)`. Otherwise, the resulting image is smaller for some reason.

**Comment by espadrine** [comment]
A few random pieces of information mostly for future me:

- Looks like it doesn't use multiple cores; it only has a single imagemagick process running at once. So multiple PNG images will each wait for the previous one to be processed. Not a big deal in practice, as currently our servers are single-core.
- All things taken into account, the speedup CPU-wise will be fairly small on average if not inexistant, but the memory consumption will be drastically reduced, which was the main motivation.

**Comment by espadrine** [comment]
I put it live. I [tweaked](https://github.com/badges/ServerScript/commit/917d61776d40a6006cd2735bf9ac9f3b863ae921) the installation script to acknowledge the necessity for installing ImageMagick.

The font is indeed slightly inaccurate, but it looks ok. Overall, pretty good result. Thanks for the good work!

**Pull request closed without merging**